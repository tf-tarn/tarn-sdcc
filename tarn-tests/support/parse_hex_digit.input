00
01
02
03
04
05
06
07
08
09
0a
0b
0c
0d
0f0a0f
ff
ff
ff
