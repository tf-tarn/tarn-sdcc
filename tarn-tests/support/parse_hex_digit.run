./sim/sim -q -s 1000 $program_file > $output
