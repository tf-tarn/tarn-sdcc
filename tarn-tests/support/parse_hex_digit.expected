Copy section of size 6 to 0x4000
Copy section of size 824 to 0x4006
Copy section of size 39 to 0x433e
00
01
02
03
04
05
06
07
08
09
0a
0b
0c
0d
0f0a0f
ff
ff
ff
